src/main/java/org/broadinstitute/consent/http/models/LibraryCard.java

```diff
@@ -31,7 +31,7 @@ public class LibraryCard {
 
   private Institution institution;
 
-  private List<Integer> daaIds;
+  private List<Integer> daaIds = new ArrayList<>();
```

Comment:
I'd recommend initializing this instance variable in the constructor instead, like `createDate` is doing.